index.html

```diff
@@ -239,6 +239,9 @@ <h4 class="mb-0"><strong>First Contribution Hacktoberfest 2022 List</strong></h4
         </div>
       </div>
     </div>
+    <footer style="background-color:black ; color: white;text-align: center;padding: 5px;">
```

Comment:
Hey @shivanidotcom 
Add the css class to footer and then define that class in css file. 
No need for inline styles.